### ISSUE TITLE
CLIPper fails with no quality scores

### ISSUE THREAD

**Issue by mlovci** [YeoLab/clipper#33]
Clipper fails ungracefully when .bam files don't have quality scores.

**Comment by ShanSabri** [comment]
I'm having the same issue. Any thoughts?

**Comment by gpratt** [comment]
Sorry for the late reply.  Have you had any luck hacking in quality scores
into your bam file?  I think that would be the best way around this issue.

Gabriel Pratt
Bioinformatics Graduate Student, Yeo Lab
University of California San Diego

**Comment by ShanSabri** [comment]
Yes. If I impute quality scores, then clipper runs successfully.
